.storybook/webpack.config.js

```diff
@@ -20,28 +20,28 @@ function getBuildOuptutPathList(buildOutputRelativePath) {
 
 // Mock ROOT_GEN_DIR as 'gen' - it will be replaced with an array from
 // getBuildOutputPathList.
-process.env.ROOT_GEN_DIR = 'gen'
+const potentialGenFolders = getBuildOuptutPathList('gen')
+process.env.ROOT_GEN_DIR = potentialGenFolders
+  .filter(a => fs.existsSync(a))
+  .sort((a, b) => fs.statSync(a).mtime - fs.statSync(b).mtime)[0]
+
+  // Fallback to the first folder in the list, so we get sensible errors.
+  ?? potentialGenFolders[0]
```

Comment:
If none of them exist, shouldn't we just throw an error here?

Reply:
That might be sensible - I'll update.

Reply:
I think that would be sensible so people know they can't generate Storybook without a build first.